### ISSUE TITLE
Payment account names are confusing

### ISSUE THREAD

**Issue by Hanawa02** [eckon/money-tracker-backend#4]
Please use `from` instead of payer and `to` instead of lender. This will make it clear who is paying who.

The main confusion was because the term payer is also used on cost to define who payed the cost.

![image](https://user-images.githubusercontent.com/11237366/206861082-44529543-5488-46a9-817e-93221cd0ff42.png)

**Comment by eckon** [comment]
yes, I want to change that in general. using `from` and `to` might be a good replacement
